Dockerfile

```
@@ -0,0 +1,20 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && \
+    apt-get install python3 python3-pip -y && \
+    groupadd -g 1234 notroot && \
+    useradd -m -u 1234 -g notroot notroot
+
+USER notroot
+
+WORKDIR /app
+
+COPY requirements.txt .
```

Here you need to change the ownership of the copied files:
```suggestion
COPY --chown=notroot:notroot requirements.txt .
```